cypress/unit/store-getter-getCategoricalJsonOutput.cy.js

```diff
@@ -0,0 +1,106 @@
+import { getters } from "~/store";
+
+const category = "category1";
+const columnName = "column1";
+let store = {
+
+    getters: getters,
+
+    state: {
+
+        categoricalOptions: {
+
+            category1: [
+
+                { label: "annotatedValue1", identifier: "https://example.org/option_1"},
+                { label: "annotatedValue2", identifier: "https://example.org/option_2"}
+            ]
+        },
+        categories: {
+            "category1": {
+
+                componentName: "annot-categorical",
+                explanation: "This is an explanation for how to annotate category1.",
+                identifier: "nb:hasCategory1"
+            }
+        },
+        columnToCategoryMap: { column1: "category1" },
+        dataDictionary: {
+
+            annotated: {
+
+                column1: {
+
+                    valueMap: { "rawValue1": "https://example.org/option_1" },
+                    missingValues: {0: "missing"}
```

Comment:
```suggestion
                    missingValues: ["missing"]
```
`missingValues` is an array, see e.g. https://github.com/neurobagel/annotation_tool/blob/784b974636b8f72740d4ca98138f153d81e70a61/cypress/unit/store-mutation-changeMissingStatus.cy.js